### ISSUE TITLE
$Bin triggers ProhibitUnusedImport when it only appears in a subsequent use

### ISSUE THREAD

**Issue by oalders** [gugod/Perl-Critic-TooMuchCode#19]
The following code will trigger the policy:

```perl
use FindBin qw( $Bin );                                                                                                                                                                                                                                                       
use lib "$Bin/../../../../lib";
```

**Comment by gugod** [comment]
A brief triage suggest that interpolated variables are not gathered as usages. 

On this alternative version it works as expected (reports no violations)

```perl
use FindBin qw( $Bin );                                                                                                                                                                                                                                                       
use lib $Bin . "/../../../../lib";
```

Test cases are recorded here: https://github.com/gugod/Perl-Critic-TooMuchCode/commit/da343da92fb8a32e586bf48234a19a08eec6a257

**Comment by oalders** [comment]
I jumped through a lot of hoops to try to catch most of these cases in `perlimports`: https://metacpan.org/release/OALDERS/App-perlimports-0.000017/source/lib/App/perlimports/Document.pm#L563-632

**Comment by gugod** [comment]
Thanks @oalders. The changes included in PR #27 seems good enough and that could be released. It looks like the code from `perlimports` also catchs variables in the form of `${VAR}` -- which is not catched by ProhibitUnusedImport. Perhaps this question itself deserves a place in PPI / PPIx::Utitis  :)

**Comment by gugod** [comment]
fixed in 0.16

**Comment by oalders** [comment]
Thanks @gugod!
